packages/usa-input-prefix-suffix/src/styles/_usa-input-prefix-suffix.scss

```
@@ -21,28 +24,52 @@
     @include u-border($theme-input-state-border-width, "success");
   }
 
+  &--error,
+  &--success {
+    & input:focus {
+      outline-offset: units($theme-input-state-border-width);
+    }
+  }
+
   input {
+    @include u-padding-right($prefix_suffix_offset);
     border: 0;
     height: 100%;
     margin-top: 0;
     min-width: 0;
     width: 100%;
 
-    &:focus {
-      outline: none;
+    &:disabled + .usa-input-suffix,
+    &[aria-disabled="true"] + .usa-input-suffix {
+        @include u-disabled;
     }
   }
 }
 
+.usa-input-prefix + input {
+  @include u-padding-left($prefix_suffix_offset);
+  @include u-padding-right(1);
```

Could this be defined with `$prefix_suffix_padding`?
```suggestion
  @include u-padding-right($prefix_suffix_padding);
```

---

Resolved in https://github.com/uswds/uswds/pull/5004/commits/5db72ab757a65629f44993a4b6b99d08e58f0b11